src/VssCommandProcessor.cpp

```diff
@@ -65,82 +65,7 @@ VssCommandProcessor::~VssCommandProcessor() {
 #endif
 }
 
-string VssCommandProcessor::processGet(WsChannel &channel,
-                                       const string & request_id, 
-                                       const string & path) {
-  logger->Log(LogLevel::VERBOSE, "GET :: path received from client = " + path);
-  jsoncons::json res;
-  try {
-    res = database->getSignal(channel, path);
-  } catch (noPermissionException &nopermission) {
-    logger->Log(LogLevel::ERROR, string(nopermission.what()));
-    return JsonResponses::noAccess(request_id, "get", nopermission.what());
-  } catch (std::exception &e) {
-    logger->Log(LogLevel::ERROR, "Unhandled error: " + string(e.what()));
-    return JsonResponses::malFormedRequest(request_id, "get", string("Unhandled error: ") + e.what());
-  }
-
-  if (!res.contains("value")) {
-    return JsonResponses::pathNotFound(request_id, "get", path);
-  } else {
-    res["action"] = "get";
-    res["requestId"] = request_id;
-    res["timestamp"] = JsonResponses::getTimeStamp();
-    stringstream ss;
-    ss << pretty_print(res);
-    return ss.str();
-  }
-}
-
-string VssCommandProcessor::processSet(WsChannel &channel,
```

Comment:
where is the code moving? not necessary any more? but https://github.com/eclipse/kuksa.val/blob/7ddad8da5c3f208592f64d5e31041cafcc6d898d/include/VssCommandProcessor.hpp#L46 declaration is still there

Reply:
Good catch. Not needed (I hope), superseded by the VSSPath variant